{
  "uid" : "69946c50982550f5",
  "name" : "test_registration_negative[new email and empty first name--1928-autotest_N@1726582210.8004487.com-password-password-error_message3]",
  "fullName" : "tests.registration_page_test.TestRegistrationPage#test_registration_negative",
  "historyId" : "0109e9ddbeea7be63b69bc2c6906b063",
  "time" : {
    "start" : 1726582366130,
    "stop" : 1726582366130,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_SSL_SERVER_CERT_BAD_FORMAT\n  (Session info: chrome=128.0.6613.119)\nStacktrace:\n#0 0x55c4d478d86a <unknown>\n#1 0x55c4d445be50 <unknown>\n#2 0x55c4d4454011 <unknown>\n#3 0x55c4d44440e9 <unknown>\n#4 0x55c4d4445e54 <unknown>\n#5 0x55c4d444439d <unknown>\n#6 0x55c4d4443c1c <unknown>\n#7 0x55c4d4443b0d <unknown>\n#8 0x55c4d4441b11 <unknown>\n#9 0x55c4d44421aa <unknown>\n#10 0x55c4d445e771 <unknown>\n#11 0x55c4d44ef515 <unknown>\n#12 0x55c4d44d0492 <unknown>\n#13 0x55c4d44ee9c6 <unknown>\n#14 0x55c4d44d0233 <unknown>\n#15 0x55c4d449e093 <unknown>\n#16 0x55c4d449f09e <unknown>\n#17 0x55c4d4754acb <unknown>\n#18 0x55c4d4758a81 <unknown>\n#19 0x55c4d4740695 <unknown>\n#20 0x55c4d47595f2 <unknown>\n#21 0x55c4d472586f <unknown>\n#22 0x55c4d477c668 <unknown>\n#23 0x55c4d477c832 <unknown>\n#24 0x55c4d478c65c <unknown>\n#25 0x7f2a78a94ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"ac14a34e0dd715a3da06a6d770d6bcff\")>\n\n    @pytest.fixture()\n    @allure.step(f'Open page: {MainPageLinks.URL_MAIN_PAGE}')\n    def main_page_open(driver):\n>       driver.get(MainPageLinks.URL_MAIN_PAGE)\n\ntests/conftest.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f7d7755fbc0>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_SSL_SERVER_CERT_BAD_FOR...\\\\n#23 0x55c4d477c832 \\\\u003Cunknown>\\\\n#24 0x55c4d478c65c \\\\u003Cunknown>\\\\n#25 0x7f2a78a94ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_SSL_SERVER_CERT_BAD_FORMAT\nE         (Session info: chrome=128.0.6613.119)\nE       Stacktrace:\nE       #0 0x55c4d478d86a <unknown>\nE       #1 0x55c4d445be50 <unknown>\nE       #2 0x55c4d4454011 <unknown>\nE       #3 0x55c4d44440e9 <unknown>\nE       #4 0x55c4d4445e54 <unknown>\nE       #5 0x55c4d444439d <unknown>\nE       #6 0x55c4d4443c1c <unknown>\nE       #7 0x55c4d4443b0d <unknown>\nE       #8 0x55c4d4441b11 <unknown>\nE       #9 0x55c4d44421aa <unknown>\nE       #10 0x55c4d445e771 <unknown>\nE       #11 0x55c4d44ef515 <unknown>\nE       #12 0x55c4d44d0492 <unknown>\nE       #13 0x55c4d44ee9c6 <unknown>\nE       #14 0x55c4d44d0233 <unknown>\nE       #15 0x55c4d449e093 <unknown>\nE       #16 0x55c4d449f09e <unknown>\nE       #17 0x55c4d4754acb <unknown>\nE       #18 0x55c4d4758a81 <unknown>\nE       #19 0x55c4d4740695 <unknown>\nE       #20 0x55c4d47595f2 <unknown>\nE       #21 0x55c4d472586f <unknown>\nE       #22 0x55c4d477c668 <unknown>\nE       #23 0x55c4d477c832 <unknown>\nE       #24 0x55c4d478c65c <unknown>\nE       #25 0x7f2a78a94ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1726582211375,
      "stop" : 1726582211375,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "main_page_open",
    "time" : {
      "start" : 1726582366900,
      "stop" : 1726582368108,
      "duration" : 1208
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_SSL_SERVER_CERT_BAD_FORMAT\n  (Session info: chrome=128.0.6613.119)\nStacktrace:\n#0 0x55c4d478d86a <unknown>\n#1 0x55c4d445be50 <unknown>\n#2 0x55c4d4454011 <unknown>\n#3 0x55c4d44440e9 <unknown>\n#4 0x55c4d4445e54 <unknown>\n#5 0x55c4d444439d <unknown>\n#6 0x55c4d4443c1c <unknown>\n#7 0x55c4d4443b0d <unknown>\n#8 0x55c4d4441b11 <unknown>\n#9 0x55c4d44421aa <unknown>\n#10 0x55c4d445e771 <unknown>\n#11 0x55c4d44ef515 <unknown>\n#12 0x55c4d44d0492 <unknown>\n#13 0x55c4d44ee9c6 <unknown>\n#14 0x55c4d44d0233 <unknown>\n#15 0x55c4d449e093 <unknown>\n#16 0x55c4d449f09e <unknown>\n#17 0x55c4d4754acb <unknown>\n#18 0x55c4d4758a81 <unknown>\n#19 0x55c4d4740695 <unknown>\n#20 0x55c4d47595f2 <unknown>\n#21 0x55c4d472586f <unknown>\n#22 0x55c4d477c668 <unknown>\n#23 0x55c4d477c832 <unknown>\n#24 0x55c4d478c65c <unknown>\n#25 0x7f2a78a94ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Open page: https://brainup.site/",
      "time" : {
        "start" : 1726582366900,
        "stop" : 1726582368107,
        "duration" : 1207
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_SSL_SERVER_CERT_BAD_FORMAT\n  (Session info: chrome=128.0.6613.119)\nStacktrace:\n#0 0x55c4d478d86a <unknown>\n#1 0x55c4d445be50 <unknown>\n#2 0x55c4d4454011 <unknown>\n#3 0x55c4d44440e9 <unknown>\n#4 0x55c4d4445e54 <unknown>\n#5 0x55c4d444439d <unknown>\n#6 0x55c4d4443c1c <unknown>\n#7 0x55c4d4443b0d <unknown>\n#8 0x55c4d4441b11 <unknown>\n#9 0x55c4d44421aa <unknown>\n#10 0x55c4d445e771 <unknown>\n#11 0x55c4d44ef515 <unknown>\n#12 0x55c4d44d0492 <unknown>\n#13 0x55c4d44ee9c6 <unknown>\n#14 0x55c4d44d0233 <unknown>\n#15 0x55c4d449e093 <unknown>\n#16 0x55c4d449f09e <unknown>\n#17 0x55c4d4754acb <unknown>\n#18 0x55c4d4758a81 <unknown>\n#19 0x55c4d4740695 <unknown>\n#20 0x55c4d47595f2 <unknown>\n#21 0x55c4d472586f <unknown>\n#22 0x55c4d477c668 <unknown>\n#23 0x55c4d477c832 <unknown>\n#24 0x55c4d478c65c <unknown>\n#25 0x7f2a78a94ac3 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.5/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"ac14a34e0dd715a3da06a6d770d6bcff\")>"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  }, {
    "name" : "driver",
    "time" : {
      "start" : 1726582366130,
      "stop" : 1726582366900,
      "duration" : 770
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1726582368139,
      "stop" : 1726582368202,
      "duration" : 63
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Test Registration Page"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "registration_page_test"
  }, {
    "name" : "subSuite",
    "value" : "TestRegistrationPage"
  }, {
    "name" : "host",
    "value" : "fv-az1535-406"
  }, {
    "name" : "thread",
    "value" : "2301-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.registration_page_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "birthday",
    "value" : "1928"
  }, {
    "name" : "confirm_password",
    "value" : "'password'"
  }, {
    "name" : "email",
    "value" : "'autotest_N@1726582210.8004487.com'"
  }, {
    "name" : "error_message",
    "value" : "['Empty LastName', 'Фамилия не Указана']"
  }, {
    "name" : "first_name",
    "value" : "''"
  }, {
    "name" : "password",
    "value" : "'password'"
  }, {
    "name" : "title",
    "value" : "'new email and empty first name'"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "69946c50982550f5.json",
  "parameterValues" : [ "1928", "'password'", "'autotest_N@1726582210.8004487.com'", "['Empty LastName', 'Фамилия не Указана']", "''", "'password'", "'new email and empty first name'" ]
}